_docs/_latest/configure/general/non-org-root.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Running Forseti from Non-Organization Root
+order: 100
+---
+
+# {{ page.title }}
+
+By default, Forseti is designed to be installed with complete
+organization access, and run with the organization as the root node in the
+resource hierarchy.
+
+But, you also have the option to run Forseti even if you only own a subset
+of resources, such as a specific folder, or projects that are directly under
+the organization. Inventory and Scanner will be supported for use on these
+subset of resources, but Explain will not be supported.
```

Comment:
What about data model? probably would me worth mentioning if raw data will be converted to structured data when organization is a folder or projects.